tools/build.py

```diff
@@ -293,11 +299,10 @@ def build_cmake_args(options, for_jerry=False):
 
     # external include dir
     include_dirs = []
-    if options.target_os == 'nuttx':
-        include_dirs.append('%s/include' % options.nuttx_home)
+    if options.target_os == 'nuttx' and options.sysroot:
```

Comment:
Yes, this is taken care of by the adjust_options function.
Yet this function (build_cmake_args) is unrelated to adjust_options, and shouldn't depend on the latter having been called before.

I could think of putting an assert equivalent here, but that would not be in line with the rest of the file.